assets/maps/prefabs/space/prefab_hunting.dmm

```diff
@@ -0,0 +1,502 @@
+/area/prefab/huntingsupplies)
+"e" = (
+/obj/machinery/light/small/floor/netural,
+/turf/simulated/floor/engine/caution/east,
+/area/prefab/huntingsupplies)
+"f" = (
+/obj/rack,
+/obj/item/ammo/bullets/a12/bird,
+/turf/simulated/floor/wood/three,
+/area/prefab/huntingsupplies)
+"h" = (
+/obj/machinery/door/airlock/pyro{
+	requires_power = 0;
+	dir = 8
+	},
+/obj/mapping_helper/airlock/bolter,
+/turf/simulated/floor/dojo,
+/area/prefab/huntingsupplies)
+"k" = (
+/obj/stool/chair/green,
+/mob/living/carbon/human/normal/assistant{
+	name = "Boxitt"
+	},
+/turf/simulated/floor/wood/three,
+/area/prefab/huntingsupplies)
+"l" = (
+/obj/mapping_helper/airlock/welder,
+/obj/machinery/door/airlock/pyro/weapons{
+	dir = 4
+	},
+/obj/mapping_helper/access/hos,
+/turf/simulated/floor/plating,
+/area/prefab/huntingsupplies)
+"m" = (
+/turf/variableTurf/wall,
+/area/allowGenerate)
+"n" = (
+/turf/variableTurf/clear,
+/area/allowGenerate)
+"o" = (
+/mob/living/carbon/human/normal/assistant{
+	name = "Big Bill Hell"
+	},
+/turf/simulated/floor/wood/three,
+/area/prefab/huntingsupplies)
+"p" = (
+/turf/simulated/wall/auto/reinforced/jen/dark4,
+/area/prefab/huntingsupplies)
+"q" = (
+/obj/rack,
+/turf/simulated/floor/wood/three,
+/area/prefab/huntingsupplies)
+"r" = (
+/obj/rack,
+/obj/item/clothing/suit/armor/gang,
+/turf/simulated/floor/wood/three,
+/area/prefab/huntingsupplies)
+"s" = (
+/obj/machinery/light/incandescent/blueish{
+	dir = 8
+	},
+/turf/simulated/floor/wood/three,
+/area/prefab/huntingsupplies)
+"t" = (
+/obj/decal/stripe_delivery,
+/obj/table/reinforced/industrial/auto,
+/obj/item/pipebomb/frame{
+	pixel_y = 5
+	},
+/obj/item/cable_coil/yellow,
+/turf/simulated/floor/dojo,
+/area/prefab/huntingsupplies)
+"u" = (
+/obj/machinery/light/small/floor/warm,
+/turf/simulated/floor/dojo,
+/area/prefab/huntingsupplies)
+"v" = (
+/obj/machinery/door/airlock/pyro/glass/mining{
+	requires_power = 0
+	},
+/turf/simulated/floor/wood/three,
+/area/prefab/huntingsupplies)
+"w" = (
+/obj/rack,
+/obj/item/clothing/head/NTberet,
+/turf/simulated/floor/wood/three,
+/area/prefab/huntingsupplies)
+"x" = (
+/obj/decoration/vent/vent3{
+	pixel_y = 28
+	},
+/turf/simulated/floor/wood/three,
+/area/prefab/huntingsupplies)
+"y" = (
+/turf/simulated/floor/dojo,
+/area/prefab/huntingsupplies)
+"z" = (
+/obj/machinery/light/incandescent/blueish{
+	dir = 4
+	},
+/turf/simulated/floor/wood/three,
+/area/prefab/huntingsupplies)
+"A" = (
+/obj/random_pod_spawner/random_putt_spawner,
+/turf/simulated/floor/engine,
+/area/prefab/huntingsupplies)
+"C" = (
+/turf/simulated/floor/engine,
+/area/prefab/huntingsupplies)
+"D" = (
+/obj/machinery/light/small/blueish,
+/turf/simulated/floor/wood/three,
+/area/prefab/huntingsupplies)
+"E" = (
+/obj/decal/poster/flag/progressive{
+	pixel_y = 32
+	},
+/turf/simulated/floor/wood/three,
+/area/prefab/huntingsupplies)
+"F" = (
+/obj/item/storage/secure/ssafe{
+	pixel_y = 32;
+	spawn_contents = list(/obj/item/gun/kinetic/sawnoff/birdshot);
```

Comment:
Gang gear, also highly likely this will get abused. 

Additionally the way this safe is spawned, at least in my testing, it didn't actually set any code and the gun didn't spawn inside of it. But that might be because of debug flags, not sure.

Reply:
i added more variableturf walls around it, is that good?